src/d3a/models/market/blockchain_utils.py

```diff
@@ -28,16 +25,20 @@
 log = getLogger(__name__)
 
 
-BC_NUM_FACTOR = 10 ** 10
-BOB_ADDRESS = "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty"
-ALICE_ADDRESS = "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
+BC_NUM_FACTOR = 10 ** 12
+BOB_STASH_ADDRESS = "5HpG9w8EBLe5XCrbczpwq5TSXvedjrBGCwqxK1iQ7qUsSWFc"
+ALICE_STASH_ADDRESS = "5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY"
```

Comment:
May be add a module `blockhain.constants.py` and move all associated constants there

Reply:
Makes sense, I will bring them out

Reply:
Added commit https://github.com/gridsingularity/d3a/pull/1033/commits/13ba9779d19c47c0ec2e7ce9816df5f209e62f21